src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx

```diff
@@ -14,21 +14,37 @@ const OccasionsType = (props) => {
         {props.icon}
       </Container>
       <Container className="no-occasions__message">
-        {props.message}
+        {renderMessageText(props)}
       </Container>
     </Container>
   );
 };
 
+const renderMessageText = (props) => {
+  if (props.countType === TIME_PERIOD.TODAY) {
+    return (
+      <>
+        {NO_OCCASION_TYPE_MESSAGE.MESSAGE_FIRST} {props.message} {props.countType}{NO_OCCASION_TYPE_MESSAGE.MESSAGE_END}
+      </>
+    );
+  } else if (props.countType !== TIME_PERIOD.TODAY) {
+    return (
+      <>
+        {NO_OCCASION_TYPE_MESSAGE.MESSAGE_FIRST} {props.message} {NO_OCCASION_TYPE_MESSAGE.MESSAGE_SECOND} {props.countType}{NO_OCCASION_TYPE_MESSAGE.MESSAGE_END}
+      </>
+    );
+  }
+};
+
 const renderNoOccasionContent = (props) => {
-  const { members = [], occasionType } = props;
+  const { members = [], occasionType, countType } = props;
     if (occasionType === OCCASION_TYPE.BIRTHDAY) {
       return (
-        <OccasionsType icon={<IconCake />} message={NO_OCCASION_TYPE_MESSAGE.BIRTHDAY_MESSAGE} iconClass="birthday-icon" />
+        <OccasionsType icon={<IconCake />} message={OCCASION_TYPE.BIRTHDAYS} countType={countType} iconClass="birthday-icon" />
```

Comment:
Why are we passing OCCASION_TYPE.ANNIVERSARIES in message??
```suggestion
        <OccasionsType icon={<IconCake />} timePeriod={timePeriod} iconClass="birthday-icon" />
```

Reply:
For finding the type of occasion birthday or anniversary